IO.Eventuate.Tram.IntegrationTests/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     - mssql
     environment:
       TRAM_DB_SERVER: "mssql"
-      TRAM_SA_PASSWORD: "TestPa$$word"
+      TRAM_SA_PASSWORD: "TestPa55word"
```

Comment:
Just because?

Reply:
The `$$` was causing some issues and was actually resulting in the SQL password being set to "TestPa$word" instead of "TestPa$$word". Probably could have been fixed by added some form of escape character, but I opted to just avoid the `$` character.